css-view-transitions-2/Overview.bs

```diff
@@ -473,7 +488,7 @@ Note: as per default behavior, the ''@view-transition'' rule can be nested insid
 
 		1. Let |viewTransition| be the result of running the [=method steps=] for {{Document/startViewTransition(updateCallback)}} given |updateCallback|.
 
-		1. If |callbackOptions| is a {{StartViewTransitionOptions}}, then set |viewTransition|'s [=ViewTransition/active types=] to |callbackOptions|'s {{StartViewTransitionOptions/type}}.
+		1. If |callbackOptions| is a {{StartViewTransitionOptions}}, then [=list/extend=] |viewTransition|'s [=ViewTransition/active types=] with |callbackOptions|'s {{StartViewTransitionOptions/type}}.
```

Comment:
nit: At every instance active types is updated, its value is overwritten completely instead of appending. Can we say that instead of extend to make that obvious?

Reply:
It's the proper way to copy a list into an existing `set` (the token set). How would you reword it?

Reply:
Actually let me try something.